ghcup-vanilla-0.0.8.yaml

```diff
@@ -4226,6 +4226,128 @@ ghcupDownloads:
         dlHash: dd91c6b65bc4b424de273072fb6ae8782a6efa861cf36445cc88fc383455b209
         dlSubdir: ghc-9.6.6/testsuite
         dlUri: https://downloads.haskell.org/~ghc/9.6.6/ghc-9.6.6-testsuite.tar.xz
+    9.6.7:
+      viArch:
+        A_32:
+          Linux_Debian:
+            <10: &ghc967-i386-linux-deb9
+              dlHash: a1f59f25d6f1accf3bb521d6b623c8a062313baaa1679dd304437bbc381dce51
+              dlOutput: ghc-9.6.7-i386-linux-deb9.tar.xz
```

Comment:
If `dlOutput` matches the url, we don't want it, because it's just noise and increases file size.